src/components/dropdownSearch/DropdownSearch.tsx

```diff
@@ -99,8 +99,8 @@ export class DropdownSearch extends React.Component<IDropdownSearchProps, void>
           : value;
 
         return <li key={option.value}
-                   className={liClasses}
-                   title={value}>
+          className={liClasses}
+          title={value}>
```

Comment:
I changed this part on my PR! I also added Parenthesis around JSX object. You should revert those two lines because they are not related to this PR.